lib/action_controller/serialization.rb

```diff
@@ -20,6 +20,10 @@ def serialization_scope
     end
 
     def get_serializer(resource, options = {})
+      unless options[:serializer] || options[:each_serializer] || ActiveModel::Serializer.config.automatic_lookup
+        return resource
+      end
+
       if !use_adapter?
```

Comment:
(or add `ActiveModel::Serializer.config.automatic_lookup` as `config.implicit_serializers = false` which would used probably in the serializable resource